test/inspector/test-on-off.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const helper = require('./inspector-helper.js');
+
+function testResume(session) {
+  session.sendCommandsAndExpectClose([
+    { 'method': 'Runtime.runIfWaitingForDebugger' }
+  ]).expectStderrOutput('Debugger listening on');
+}
+
+function runTests(harness) {
+  harness
+    .runFrontendSession([
+      testResume,
+    ]).expectShutDown(0);
+}
+
+const script = 'process._debugEnd();' +
```

Comment:
1. Complex script - I noticed that starting inspector after it had been stopped crashed the Node, so some test coverage definitely helps.
2. _debugEnd - the goal here is to ensure that inspector can cleanly stop. There were some genuine bugs and race conditions that need to be addressed.